backend/src/apiserver/config/sample_config.json

```diff
@@ -1,32 +1,32 @@
-    "description":"A pipeline that downloads two messages in parallel and prints the concatenated result. For source code, refer to https://github.com/kubeflow/pipelines/blob/master/samples/core/parallel_join/parallel_join.py",
-    "file":"/samples/core/parallel_join/parallel_join.py.tar.gz"
+    "name": "[Sample] Basic - Parallel execution",
+    "description": "A pipeline that downloads two messages in parallel and prints the concatenated result. [source code](https://github.com/kubeflow/pipelines/blob/master/samples/core/parallel_join/parallel_join.py)",
+    "file": "/samples/core/parallel_join/parallel_join.py.tar.gz"
   },
   {
-    "name":"[Sample] Basic - Conditional execution",
-    "description":"A pipeline shows how to use dsl.Condition. For source code, refer to https://github.com/kubeflow/pipelines/blob/master/samples/core/condition/condition.py",
-    "file":"/samples/core/condition/condition.py.tar.gz"
+    "name": "[Sample] Basic - Conditional execution",
+    "description": "A pipeline shows how to use dsl.Condition. [source code](https://github.com/kubeflow/pipelines/blob/master/samples/core/condition/condition.py)",
+    "file": "/samples/core/condition/condition.py.tar.gz"
   },
   {
-    "name":"[Sample] Basic - Exit Handler",
-    "description":"A pipeline that downloads a message and prints it out. Exit Handler will run at the end. For source code, refer to https://github.com/kubeflow/pipelines/blob/master/samples/core/exit_handler/exit_handler.py",
-    "file":"/samples/core/exit_handler/exit_handler.py.tar.gz"
+    "name": "[Sample] Basic - Exit Handler",
+    "description": "A pipeline that downloads a message and prints it out. Exit Handler will run at the end. [source code](https://github.com/kubeflow/pipelines/blob/master/samples/core/exit_handler/exit_handler.py)",
+    "file": "/samples/core/exit_handler/exit_handler.py.tar.gz"
   },
   {
-    "name":"[Sample] ML - TFX - Taxi Tip Prediction Model Trainer",
-    "description":"Example pipeline that does classification with model analysis based on a public tax cab BigQuery dataset. GCS permission is required. For guideline of setting up permission after deployment, please see https://github.com/kubeflow/pipelines/blob/master/manifests/gcp_marketplace/guide.md#gcp-service-account-credentials. For source code, refer to https://github.com/kubeflow/pipelines/tree/master/samples/contrib/parameterized_tfx_oss",
-    "file":"/samples/contrib/parameterized_tfx_oss/parameterized_tfx_oss.tar.gz"
+    "name": "[Sample] ML - TFX - Taxi Tip Prediction Model Trainer",
+    "description": "[Permission requirements](https://github.com/kubeflow/pipelines/blob/master/manifests/gcp_marketplace/guide.md#gcp-service-account-credentials). Example pipeline that does classification with model analysis based on a public tax cab BigQuery dataset. GCS permission is required. [source code](https://github.com/kubeflow/pipelines/tree/master/samples/contrib/parameterized_tfx_oss)",
```

Comment:
This should point to TFX sample README file which explains service account permission settings. The link you have here is general service account creation, but specific to this sample. The TFX sample README references this link, but with additional service account role instructions.

Reply:
Thanks for pointing this out. Sent #2422 to fix.

Reply:
SG, thanks for the fix!

Reply:
Updated link